packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -2240,7 +2236,7 @@ def background_round_tx_type(self) -> Optional[str]:
         """Returns the allowed transaction type for background round."""
         if self.is_termination_set:
             return cast(AppState, self.background_round_cls).allowed_tx_type
-        return None
+        return None  # pragma: no cover
```

Comment:
should have never been exposed in the first place. Used internally for `check_transaction` and `process_transaction` only where it can never return `None`.

https://github.com/valory-xyz/open-autonomy/blob/16041619aa63ecad6beec6cc2e4f872d2223748a/packages/valory/skills/abstract_round_abci/base.py#L2234-L2239

Reply:
We have a bunch of improvements to do on this specific one. I am adding this to the ticket that contains the rest.

Reply:
https://github.com/valory-xyz/open-autonomy/issues/1505#issuecomment-1356180888